reference/api/search.md

```diff
@@ -525,6 +575,45 @@ The highlighted version of the text would then be found in the `_formatted` obje
 }
 ```
 
+### Highlight tags
+
+**Parameters**: `highlightPreTag` and `highlightPostTag`
+**Expected value**: a string
+**Default value**: `"<em>"` and `"</em>"`
+
+`highlightPreTag` and `highlightPostTag` configure what string of text appears respectively before and after a word highlighted by `attributesToHighlight`. If `attributesToHighlight` has not been specified, `highlightPreTag` and `highlightPostTag` have no effect on the returned search results.
+
+It is possible to use `highlightPreTag` and `highlightPostTag` to enclose terms between any string of text, not only HTML tags: `"<em>"`, `"<strong>"`, `"*"`, and `"__"` are all equally supported values.
```

Comment:
The `tag` bit is exactly why I wanted to include this paragraph. It would make no sense for Meilisearch to check for valid HTML tags, but I could see myself hesitant or not even considering I could "misuse" `highlightPre/PostTag`.